src/model/filemanager.js

```diff
@@ -12,7 +12,6 @@ import response from '../helpers/response';
 class FileManager {
     constructor() {
         this.exportPath = NSTemporaryDirectory() + 'sketch-frontify/';
-        this.clearExportFolder();
```

Comment:
My guess is that this line was added here to ensure a clean export folder when FileManager is instantiated, but if multiple instances are created the export folder might get emptied at an unwanted time. We can either remove it, or make sure there is only one FileManager instance.

I'm not sure if removing the line has other unwanted consequences, like having leftovers which may create trouble in the future (if files are not deleted explicitly). Is there a way to check this?

Reply:
When I kept an eye on that the `NSTemporaryDirectory` I didn't see any leftovers, and it seems that it's only artboard exports that are saved there.

Anyway when exports are saved they replace whatever (if anything) is at the exact destination path, and uploads are made using the exact file path too, so don't see any issue there.

I'm not super confident because of how messy this upload stuff is, but that's my thinking. I guess one thing that would throw a spanner in the works is if there is indeed something else in the plugin saving to that directory, but like I said I couldn't find anything.

Reply:
I understand. Let's see if Remo can confirm. Great job for testing all this 👍

Reply:
Thanks for the review 👍

Reply:
Hi Jamie
Unless you would save your image previews there I'm sure that only exported artboards are saved in this folder. I'm basically fine with your solution as long as you are quite sure that there are no leftovers 👍